examples/show_tataratat.py

```diff
@@ -187,7 +187,9 @@
     face_showables = []
     spline_showables = []
     for f in fanfare:
-        showables = f.showable(resolutions=500, c=(255, 215, 0))
```

Comment:
I liked the kwargs, it was a nice way of keeping the old information but temporarily change the displayed information. You just passed what you wanted into show and then in the next line you could retrieve your old data-struture

Reply:
I found it to be very inconsistent since there is a dedicated way to change the show_options, these are at some point (and depending on the object the show functions is called on) overwritten or not. 

Can we talk with Jae if we want to keep supporting this? 

My thinking on kwargs-based variables in functions is that you never know what you can actually do and what happens with the options you put in. If you want to have some options it might do what you expect or maybe not. Variable chasing is also much harder to do, where did this come from?